src/aklite_client_ext.cc

```diff
@@ -278,6 +278,13 @@ bool AkliteClientExt::RebootIfRequired() {
   return false;
 }
 
+bool AkliteClientExt::isRollbackProtectionEnabled() {
+  OSTree::Sysroot::Ptr ostree_sysroot_;
+  auto ostree_sysroot = std::make_shared<OSTree::Sysroot>(client_->config.pacman);
+  auto bootloader_lite = bootloader::BootloaderLite(client_->config.bootloader, *client_->storage, ostree_sysroot);
+  return bootloader_lite.isRollbackProtectionEnabled();
```

Comment:
I would add a method to `LiteClient` similar to `LiteClient::isBootFwUpdateInProgress()`.